man/runc-exec.8.md

```diff
@@ -2,7 +2,7 @@
    runc exec - execute new process inside the container
 
 # SYNOPSIS
-   runc exec [command options] <container-id> <container command>
+   runc exec [command options] <container-id> <container command> [-- command options]
```

Comment:
no it doesnt because `p.Args = append(p.Args[:1], p.Args[2:]...)`. I think @mrunalp also pasted an example :D